src/TypeScriptInternal.ts

```diff
@@ -0,0 +1,38 @@
+import * as ts from 'typescript'
+
+// Functions in this file are based directly off corresponding functions
+// in the TypeScript codebase.
+
+export function shouldSkipAlias(node: ts.Node): boolean {
+  switch (node.kind) {
+    case ts.SyntaxKind.ImportClause:
+    case ts.SyntaxKind.ImportEqualsDeclaration:
+      // TODO: How do we test this code path?
+      return true;
+    case ts.SyntaxKind.ImportSpecifier:
+      return node.parent.kind === ts.SyntaxKind.NamedImports;
+    case ts.SyntaxKind.BindingElement:
+    case ts.SyntaxKind.VariableDeclaration:
+      // TODO: How do we test this code path?
```

Comment:
Able to put just a standard JS file alongside the source?

Reply:
That's not enough, we need to somehow create a BindingElement or VariableDeclaration that is also an alias and I don't know how to do that, because the different `SymbolFlag` values and syntax node types in the TypeScript compiler aren't documented.